frameworks/FSharp/giraffe/src/App/Stock.fs

```diff
@@ -0,0 +1,42 @@
+﻿module Stock
+
+open Giraffe
+open Dapper
+open Npgsql
+open Microsoft.AspNetCore.Http
+open Models
+open FSharp.Control.Tasks
+open System.Text
+
+let application : HttpHandler = 
+    
+    let fortunes : HttpHandler = 
+        let extra = {id = 0; message = "Additional fortune added at request time."}
+        let extra' = Seq.singleton extra
+
+        fun (_ : HttpFunc) (ctx : HttpContext) ->
+            task {
+                use conn = new NpgsqlConnection(ConnectionString)
+                let! data = conn.QueryAsync<Fortune>("SELECT id, message FROM fortune")
+
+                let view = 
+                    data 
+                    |> Seq.append extra'

```

Comment:
Using Seq module operations are poor efficiency as involve a lot of temp structures, the Custom implementation of adding extra is better (using lists), why not used here? Is it to be most "intuitive" F# implementation?

Reply:
This is probably how I would write it without thinking of perf, will replace with list bit